tasks/lib/CodeSection.js

```diff
@@ -133,5 +132,10 @@ module.exports = class CodeSection {
     }
     return startIndex;
   }
+
+  cleanUpCode(input) {
+    const encodedTemplateRegexp = /\[\[\s*\n*<span\sclass="hljs\-attr">(range|if|else|end)?(<\/span><span\sclass="hljs\-attr">)?(\.[A-Za-z]*)?<\/span>\s*\n*\]\]/g
```

Comment:
Can you move the regex declaration to a constant at the top of the file. 

* `/s*/n*` -> the /n will never match as /s matches all whitespaces
* can you simplify `<span\sclass="hljs\-attr">` to `<.*?>` ? (I'm not sure)
* can you simplify `(range|if|else|end)?` to `(.)*?` ? (I'm not sure)